### PR TITLE
Marginal space validation

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
### Description (*)

No marginal space validation added for identifier field in cms block grid in-line edit.
Magento admin path to CMS Block admin grid :
Content -> Elements -> Blocks

This validation allows us to prevent users from entering leading and trailing spaces in Identifier field using in-line edit in the Grid.

### Related Pull Requests

https://github.com/magento/magento2/pull/26261

Closing the above PR, because I accidentally deleted my forked repository and messed up the branch with unwanted commits.

### Manual testing scenarios (*)

1. Enter text with leading and trailing spaces in the Identifier field, and it will display an error message.
2. Enter text without leading and trailing spaces in the Identifier field and it won't display an error message.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ajithkumar-maragathavel** [comment]
@rogyar I have made all the changes that you requested in the below PR. Please review it.
https://github.com/magento/magento2/pull/26261
Closing the above PR, since I messed up the branch.

Thanks.

File: app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml [hunk 1]
```diff
@@ -42,4 +42,34 @@
         <actionGroup ref="SaveAndCloseCMSBlockWithSplitButtonActionGroup" stepKey="saveAndCloseAction" />
         <seeElement selector="div[data-role='grid-wrapper']" stepKey="seeGridPage" />
     </test>
-</tests>
\ No newline at end of file
+    <test name="AdminCreateCmsBlockWithMarginalSpaceTest">
```
Comment (rogyar) [inline app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml]: Hi @ajithkumar-maragathavel. Maybe I've missed this point in your previous PR but we were talking about the best practices and having one ActionGroup|Section|Test per file. As far as I see, you have moved the action group to a separate file and that's awesome. Could you do the same with the test, please? So we have a separate file with the single test in the result.

Thank you!

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml (hunk 1 above)
Comment (ajithkumar-maragathavel) [reply app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml]: @rogyar I will change the code and commit again. Thanks for reminding the best practices.
But I have one doubt regarding the best practices. I am looking at the existing MFTF code in the core. I can able to see a single test file contains multiple tests in it. 
Those files need to be refactor right?

Comment (ajithkumar-maragathavel) [reply app/code/Magento/Cms/Test/Mftf/Test/AdminCreateCmsBlockTest.xml]: @rogyar Made the changes and commit the code. Please review it.
Thanks for your suggestions.

**Comment by ajithkumar-maragathavel** [comment]
@rogyar  Updated the code as per your suggestion. Please review it.
Thanks.

**Comment by rogyar** [comment]
Awesome, thank you!

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-6888](https://jira.corp.magento.com/browse/ENGCOM-6888) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**